### ISSUE TITLE
[Link Down] Kubernetes

### ISSUE THREAD

**Issue by cbloemker** [swapagarwal/awesome-eli5#5]
The Children's Illustrated Guide to Kubernetes link is broken. Domain seems to be timing out.

**Comment by swapagarwal** [comment]
Thanks for reporting! 😇
Do you know if there's a copy hosted somewhere else? We can replace/remove the link accordingly.

**Comment by cbloemker** [comment]
Yep! Found it on [this](https://cdn.chrisshort.net/The-Illustrated-Childrens-Guide-to-Kubernetes.pdf) CDN.

**Comment by swapagarwal** [comment]
Updated! Thanks a ton @cbloemker 🎂
